armory/eval/evaluator.py

```diff
@@ -193,18 +151,28 @@ def run(
                 )
             else:
                 logger.error("Is Docker Daemon running?")
-        self._delete_tmp()
+        if interactive:
+            self._delete_tmp_config()
+
+    def _escape_config_string(self):
+        escaped_config = str(self.config).replace(" ", "").replace("'", '\\"')
```

Comment:
As per our discussion over Zoom, given that users can put whatever they want in the `adhoc` section of the config file, including keys or values containing spaces, it'd be safer and more robust to encode the config in base64

Reply:
Done.